### ISSUE TITLE
AzureDataStudioPostgresExtension_ConnectionDetails_Keyboard:There is no asterix mark provided for the server field that it is a required field.

### ISSUE THREAD

**Issue by nasc17** [microsoft/azuredatastudio#16621]
https://msdata.visualstudio.com/Database%20Systems/_workitems/edit/1281414

https://user-images.githubusercontent.com/69922333/128762310-3cf92b16-4ca1-4148-a39c-b0b84948818f.mp4

ADS
Version: 1.32.0-insider (user setup)
Commit: 991b9af19880b20f5382daf5249aa31cc919560b
Date: 2021-08-06T05:23:36.445Z
VS Code: 1.56.0
Electron: 12.0.7
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Windows_NT x64 10.0.19043

Extension
PostgreSQL 0.2.7

**Repro Steps:**
Launch the application-Azure Data Studio and install extension PostgreSQL.
Tab till new connection button and press enter.
Tab till connection type dropdown and select postgresql.
Tab till  server field .

**Actual:**
There is no asterix mark provided for the server field that it is a required field.

**Expected:**
There should be a asterix mark provided for the required fields.

**User Impact:**
Keyboard users will not be able to know which fields are required field.
